entities/Chef.java

```diff
@@ -0,0 +1,67 @@
+package entities;
+
+import java.util.List;
+
+public class Chef extends Base {
+
+    //Attributes
+
+    private String firstName;
+    private String lastName;
+    private Integer experience;
+    private List<Order> assignedOrder;
```

Comment:
This is not a good idea, because orders can be reassigned. I don't think you should handle this here. Also an order can have multiple order details, taht means that multiple chef can work on a order detail.

Let's imagine a family ask for multiple meals in the within the same order.

I think yoo just remove this for now and re-visit when needed later

Reply:
done